csharp/ql/src/Stubs/make_stubs_nuget.py

```diff
@@ -106,7 +109,7 @@ def write_csproj_prefix(ioWrapper):
 print("\n --> Generated stub files: " + rawSrcOutputDir)
 
 print("\n* Formatting files")
-helpers.run_cmd(['dotnet', 'format', rawSrcOutputDir])
+run_cmd(['dotnet', 'format', rawSrcOutputDir], workDir)
```

Comment:
spurious `workDir` arg?

Reply:
Very nice catch @hvitved! Now the directory would be printed as error message... Will fix!